tests/src/common/WskTestHelpers.scala

```diff
@@ -141,6 +141,18 @@ trait WskTestHelpers extends Matchers {
     }
 
     object CliActivation extends DefaultJsonProtocol {
+        private implicit val instantSerdes = new RootJsonFormat[Instant] {
+            def write(t: Instant) = t.toEpochMilli.toJson
+
+            def read(value: JsValue) = Try {
+                value match {
+                    case JsString(t) => Instant.parse(t)
```

Comment:
While this is nice to have, our code should never return a `JsString` here, so we should drop the serdes there?